genisys/main.py

```diff
@@ -27,17 +40,27 @@ def daemon():
     # TODO: Implement the daemon logic here
 
 def run(subcommand, args, module):
-    #netplan
-    netplan = net.Netplan(args)
+    # Config Parser
+    yamlParser = cp.YAMLParser()
```

Comment:
Got this error running it locally.
```
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/home/robert/Development/genisys/genisys/main.py", line 94, in <module>
    main()
  File "/home/robert/Development/genisys/genisys/main.py", line 91, in main
    run(args.command, args, modules)
  File "/home/robert/Development/genisys/genisys/main.py", line 44, in run
    yamlParser = cp.YAMLParser()
                 ^^^^^^^^^^^^^^^
TypeError: YAMLParser.__init__() missing 1 required positional argument: 'fileName'
```
The file name I believe should be available as `args.file`.